elements/lisk-client/cypress/support/commands.js

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
everything is commented, do we need this file?

Reply:
These are autogenerated file from cypress, kept as skeleton to be useful later.